operator/src/test/java/oracle/kubernetes/operator/webhooks/AdmissionCheckerTest.java

```diff
@@ -33,10 +46,27 @@
 import static oracle.kubernetes.operator.webhooks.AdmissionWebhookTestSetUp.createCluster;
 import static oracle.kubernetes.operator.webhooks.AdmissionWebhookTestSetUp.createDomain;
 import static oracle.kubernetes.operator.webhooks.AdmissionWebhookTestSetUp.setAuxiliaryImages;
+import static oracle.kubernetes.weblogic.domain.model.Model.DEFAULT_AUXILIARY_IMAGE_MOUNT_PATH;
+import static oracle.kubernetes.weblogic.domain.model.ServerEnvVars.LOG_HOME;
+import static oracle.kubernetes.weblogic.domain.model.ServerEnvVars.SERVER_NAME;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.junit.MatcherAssert.assertThat;
 
 class AdmissionCheckerTest {
+  private static final String BAD_SERVER_NAME_1 = "bad-ms1";
+  private static final String BAD_CLUSTER_NAME_1 = "bad-cluster1";
```

Comment:
Minor question. these names are compliant with DNS_1123. why are the variables named BAD*?